source/stroom-logs/stroomContent/Internal_Logs/AUTH_APP_LOG.Feed.7752386a-0e0d-434f-9361-213e57f40a60.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<feed>
+   <createTime>1539244810954</createTime>
```

Comment:
Can you strip createTime, createUser, updateTime, updateUser

We don't tend to leave those in the content packs

Same for other xml files below.